package.json

```diff
@@ -34,6 +34,8 @@
     "express": "^4.17.1",
     "express-healthcheck": "^0.1.0",
     "feedparser-promised": "^2.0.1",
+    "node-fetch": "^2.6.0",
```

Comment:
We're using [bent](https://github.com/mikeal/bent#readme) elsewhere, can you use it too?

Reply:
I left a comment in the description:
> I wanted to use bent, but unfortunately it only likes `200` status codes, and the GitHub API does a lot redirections (status code `301`), so I couldn't get it to work.

Reply:
Ah, good to know.  Sorry I missed your comment.  Let's stay on `node-fetch`, and consider file an issue to do after this that replaces all uses of `bent` with `node-fetch`.  I'd rather not have multiple deps for doing requests.